bundles/org.openhab.binding.atlona/README.md

```diff
@@ -1,7 +1,7 @@
 # Atlona Binding
 
 This binding integrates Atlona AT-UHD-PRO3 HdBaseT matrix switches [Atlona AT-UHD-PRO3 HdBaseT matrix switches](https://www.atlona.com) into your openHAB installation.
-The older HD model 6x6 matrix [AT-PRO3HD66M] (https://atlona.com/product/at-pro3hd66m/) is also supported.
+The older HD model 6x6 matrix [AT-PRO3HD66M] (<https://atlona.com/product/at-pro3hd66m/>) is also supported.
```

Comment:
Why add the formatting here and not on line 3 also? Arguably it looks cleaner without the full url visible in the document:
![image](https://user-images.githubusercontent.com/26532542/205942745-e5f7b86a-6ab4-4b17-a093-8b1e2545ed7a.png)

Reply:
Markdownlint interpeted this as "bare url" because of the space.

I have used some bulk edit functions for this pr and already recognized that this won't be a good idea for the bare url errors.
I had already fixed some wrong urls, but it seems that i missed this one (and probably others too).

I will hve a look again and of course a named url should be preffered. You are both right.